plugins/nodes/src/js/pages/nodes/NodeDetailHealthTab.js

```diff
@@ -13,13 +13,13 @@ class NodeDetailHealthTab extends React.Component {
       <HealthTab
         node={node}
         units={units}
-        parentRouter={this.context.router} />
+        params={this.props.params} />
     );
   }
 }
 
 NodeDetailHealthTab.contextTypes = {
-  router: React.PropTypes.func
+  router: React.PropTypes.object
```

Comment:
Any chance we can make this as well as the respective `propTypes` more specific?